.github/workflows/prod-app-2-cd.yml

```
@@ -0,0 +1,131 @@
+        required: true
+      ADMIN_TOKEN:
+        required: true
+      GCP_DRIVE_FOLDER_ID:
+        required: true
+      GCP_CREDENTIALS_PATH:
+        required: true
+
+jobs:
+  deploy-prod-app-2:
+    name: prod-app-2 서버에 배포
+    if: needs.detect-idle.outputs.idle-server == 'prod-app-2'
```

actionlint (1.7.7)

property "detect-idle" is not defined in object type {}

(expression)